### ISSUE TITLE
RFE: support for includes in vifmrc

### ISSUE THREAD

**Issue by jose1711** [vifm/vifm#345]
having vifmrc support includes would allow for easy sharing of relevant parts (keyboard configuration, look, commands) with other hosts while having host-specific features (bookmarks, marks) inside separate files. it could also bring a bit more modularity in adding/removing viewers/filetype handlers.

**Comment by xaizek** [comment]
Have you seen [`:source`](https://vifm.info/vimdoc.shtml#vifm-%3Asource) command?

**Comment by jose1711** [comment]
thank you - i somehow missed it. the question is whether it can process something like ```conf.d/*``` too though.

**Comment by xaizek** [comment]
No, it can't expand globs. And the workaround is quite awful and seems to be to generate file that contains `source something` lines and source that...

**Comment by XVilka** [comment]
I think it can be solved with creating vim-like infrastructure to autoload `*.vifm` from `~/.config/vifm/autoload/` for example.

**Comment by xaizek** [comment]
Autoloading behaviour  is more akin to Vim's `plugin/` directory, `autoload/` loads files on demand which isn't suitable here. Glob expansion seems to be more flexible, you can also avoid globs and disable files by commenting them out, unlike with `plugin/`.

Expanding globs just wasn't implemented (`fnmatch()` is missing on Windows and `FindFirstFile` has different patterns). Once it's there, extending `:source` should be quite trivial.
